contracts/DollarMintingCalculator.sol

```diff
@@ -18,9 +22,19 @@ contract DollarMintingCalculator is IDollarMintingCalculator {
     function getDollarsToMint() external view override returns (uint256) {
         TWAPOracle oracle = TWAPOracle(manager.twapOracleAddress());
         uint256 twapPrice = oracle.consult(manager.uADTokenAddress());
+        bytes16 one = (uint256(1 ether)).fromUInt();
```

Comment:
should add "require" twapPrice > one  

(currently revert in the case twapPrice < one  without reason message, due to toUint of a negative number)

Reply:
done